insights/core/__init__.py

```diff
@@ -1255,6 +1255,24 @@ def _parse_line(self, line):
                 msg_info['procname'] = info_splits[4]
         return msg_info
 
+    def get_logs_by_procname(self, proc):
+        """
+        Parameters:
+            proc(str): The process or facility that you're looking for
+
+        Returns:
+            (list): The raw syslog messages produced by that process or facility
+        """
+        _log_re = re.compile(r'^[A-Za-z]{3}\s+\d{1,2}\s\d{2}:\d{2}:\d{2}\s\S+\s(?P<procname>\S+):.*$')
```

Comment:
As you might also complain, `REGEX` is not easy to be maintained and time cost. And the `_parse_line` above has been parsed the `lines` into different parts (e.g. procname, message and timestamp), would you please to change to utilize them instead of the `regex`?

https://github.com/RedHatInsights/insights-core/blob/8dbe0c56991e5519005e84cb254ddc6580aaa34b/insights/core/__init__.py#L1255

Reply:
Sure, let me provide another commit.